### PR TITLE
chore: add initial configuration for EditorConfig

### PR DESCRIPTION
**Pull request by tenshiAMD**
This is to add the initial configuration for EditorConfig.

### CODE REVIEW

**Review by momshaddinury — CHANGES_REQUESTED** [review]
@tenshiAMD Can you explain a bit what you are doing in this PR? Why those changes are needed?

**Comment by tenshiAMD** [comment]
@dinurymomshad This is for implementing coding standards in the codebase so we will have one to follow.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
```
Comment (momshaddinury) [inline .editorconfig:7]: So how will this impact if someone uses 4 indent sizes and a tab instead of space?

**Review by momshaddinury — APPROVED** [review]

**Review by momshaddinury — COMMENTED** [review]
Is it necessary, if everyone uses dart fmt?

**Comment by tenshiAMD** [comment]
@dinurymomshad in my experience, using EditorConfig is more widely used.

**Pull request merged**